sdk/servicebus/Azure.Messaging.ServiceBus/tests/Receiver/SessionReceiverLiveTests.cs

```diff
@@ -218,7 +218,7 @@ await using (var scope = await ServiceBusScope.CreateWithQueue(enablePartitionin
                 var sessionId = "sessionId1";
                 ServiceBusSender sender = client.CreateSender(scope.QueueName);
                 using ServiceBusMessageBatch batch = await sender.CreateMessageBatchAsync();
-                IEnumerable<ServiceBusMessage> messages = ServiceBusTestUtilities.AddMessages(batch, messageCount, sessionId).AsReadOnly<ServiceBusMessage>();
+                IEnumerable<AmqpMessage> messages = ServiceBusTestUtilities.AddMessages(batch, messageCount, sessionId).AsReadOnly<AmqpMessage>();
```

Comment:
I would just leave these as using ServiceBusMessage. I think we can cover this scenario with a single test in ReceiverLiveTests.

Reply:
Unless you've got a mocked batch with different behavior, only `AmqpMessage` is being stored by the batch.

Reply:
nit: update to use new util method